client/a8c-for-agencies/sections/marketplace/wpcom-overview/wpcom-slider/index.tsx

```diff
@@ -0,0 +1,151 @@
+import classNames from 'classnames';
+import { useMemo, useRef, useState } from 'react';
+import {
+	mapOptionsToSliderOptions,
+	sliderPosToValue,
+	valueToSliderPos,
+} from '../lib/wpcom-bulk-values-utils';
+import './style.scss';
+
+export type Option = {
+	label: string;
+	value: number | string | null;
+	sub?: string;
+};
+
+type Props = {
+	className?: string;
+	options: Option[];
+	onChange?: ( value: number ) => void;
+	value: number;
+	label?: string;
+	sub?: string;
+	minimum?: number;
+};
+
+export default function A4AWPCOMSlider( {
+	className,
+	options,
+	onChange,
+	value,
+	label,
+	sub,
+	minimum = 0,
+}: Props ) {
+	const total = 204;
```

Comment:
What does the `204` value represent here? Can we sync it with the amount of slider options?

Reply:
Yeah, magic number is not great in this case. I will try to update